vocabularies/Org.OData.Core.V1.xml

```diff
@@ -457,6 +457,10 @@
         <Annotation Term="Core.Description" String="The qualified name of an action in scope." />
       </TypeDefinition>
 
+      <TypeDefinition Name="QualifiedBoundOperationName" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="The qualified name of a bound action or function in scope. The qualified function name may include trailing parentheses containing the comma-separated list of parameter types, in the order of their definition, to identify a single function overload." />

```

Comment:
While the current use of this type definition does not need to identify action overloads, a generic type definition may want to allow it.

Reply:
In order to support multiple action overloads, the type definition would need the ability to specify the binding type. However, in our intended usage, the binding type is determined by the type of the element being annotated.

We discussed in our TC meeting and felt it was simpler to restrict the type definition to be used in cases where the binding parameter type is already determined (thus, the QualifiedBoundOperationName). If, in the future, we need to be able to specify a binding parameter type to identify other action overloads (or functions bound to other types), we can add a new typedef.

We added a new long description to describe the 3 cases:
-action name to specify the single action bound to the specified type
-function name to specify all function overloads bound to the specified type
-function name followed by parameters, to indicate a single function overload bound to the specified type.